.github/workflows/build_commits.yaml

```diff
@@ -0,0 +1,42 @@
+name: build_commits
+
+
+on: [pull_request]
+
+jobs:
+  build_commits:
+    name: build commit code
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v2.1.3
+        with:
+          go-version: 1.16.3
+      - name: Cache LLVM and Clang
+        id: cache-llvm
+        uses: actions/cache@v2.1.4
+        with:
+          path: $HOME/.clang
+          key: llvm-10.0
+      - name: Install LLVM and Clang
+        uses: KyleMayes/install-llvm-action@v1.2.1
+        with:
+          version: "10.0"
+          directory: $HOME/.clang
+          cached: ${{ steps.cache-llvm.outputs.cache-hit }}    
+      - name: Install libelf
+        run: |
+          sudo apt-get install libelf-dev
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: git fetch commit code
+        run: |
+          git fetch origin pull/$PR/head:rebase-head
+          git checkout rebase-head
+        env:
+          PR: ${{ github.event.pull_request.number }}  
+      - name: build commits
+        run: git rebase --exec "make clean && make -j $(nproc) && make -C bpf build_all -j $(nproc)" rebase-head
```

Comment:
This will probably work as intended, but I wonder how involved will it be for the contributor to see which exact commit this failed on. Maybe we can add `git --no-pager log --format=%B -n 1` step if the build fails?